app/templates/settings.hbs

```diff
@@ -152,6 +152,38 @@
       {{/unless}}
     </section>
   {{/if}}
+  {{#if this.showBetaFeatures}}
+    <section class="settings-section beta_features">
+      <h2 class="small-title display-inline">
+        Beta Features
+      </h2>
+      <p>
+        These features are in beta. If you find things to be not working as expected
+        or lacking, please let us know in the <a href="https://travis-ci.community">Travis CI community forum</a>
+        or email <a href="mailto:support@travis-ci.com">support</a>.
```

Comment:
Same idea here, this could be `<a href={{config-get 'urls.support'}}>`

Reply:
done 🎉